recipes/qt/6.x.x/conanfile.py

```diff
@@ -896,6 +897,8 @@ def _create_plugin(pluginname, libname, type, requires):
                 self.cpp_info.components["qtNetwork"].frameworks.append("GSS")
                 if self.options.gui and self.options.widgets:
                     self.cpp_info.components["qtPrintSupport"].system_libs.append("cups")
+                if self.options.gui and self.options.with_vulkan:
+                    self.cpp_info.components["qtGui"].frameworks.append("MoltenVK")
```

Comment:
```suggestion
                if options.get_safe("with_vulkan"):
                    self.cpp_info.components["qtGui"].requires.append("moltenvk::moltenvk")
```

Reply:
thanks for noticing my inexact suggesion @SpaceIm. Is there a reason you replace `if self.options.gui and self.options.with_vulkan:` with `if self.options.gui and options.with_vulkan:` (removal of `self.` in front of options) ?

Reply:
No, it's a typo